src/praisonai/tests/unit/test_autogen_v4_utils.py

```
@@ -0,0 +1,363 @@
+                            'expected_output': 'Summary document'
+                        }
+                    }
+                }
+            }
+        }
+        
+        topic = "artificial intelligence"
+        
+        # Simulate the task collection logic from _run_autogen_v4
+        combined_tasks = []
+        for role, details in config['roles'].items():
```

Ruff (0.12.2)

Loop control variable `role` not used within loop body

Rename unused `role` to `_role`

(B007)